numpy/core/arrayprint.py

```diff
@@ -525,11 +530,11 @@ def array2string(a, max_line_width=None, precision=None,
                     but if every element in the array can be uniquely
                     represented with an equal number of fewer digits, use that
                     many digits for all elements.
-    legacy : boolean, optional
-        If True, enables legacy printing mode, which overrides the `sign`
-        option.  Legacy printing mode approximates numpy 1.13 print output,
-        which includes a space in the sign position of floats and different
-        behavior for 0d arrays.
+    legacy : string or `False`, optional
+        If set to the string `'1.13'` enables 1.13 legacy printing mode. This
+        approximates numpy 1.13 print output by including a space in the sign
+        position of floats and different behavior for 0d arrays. If set to
+        `False`, disables legacy mode.
 
```

Comment:
Needs `.. versionadded::`. If folks want to use this they will need to check the NumPy version.

Reply:
Might want a comment here explaining that invalid versions will be ignored for forward-compatibility too.

Reply:
Don't forget to add this to both docstrings